### PR TITLE
add multiple media links for presentations

### PR DESCRIPTION
**Pull request by elmiko**
This change adds the feature for having multiple presentation urls in a
presentation entry.

changes
* update community template to process multiple urls
* update community template to add some spacing between links
* update all presentations with new link format
* update presentation template

### CODE REVIEW

**Comment by elmiko** [comment]
inspired by suggestions in #277

**Comment by erikerlandson** [comment]
This seems fine to me - is there an easy way to handle two cases: a single url or an array? that would make the pr much smaller

**Comment by elmiko** [comment]
i'll look into it, although i seem to recall that this didn't work before and putting some sort of type detection logic in the template just seems like overkill

**Comment by willb** [comment]
In general I can see a case for multiple media attachments but not necessarily to solve this problem. Why not just add the one talk we have with two media links as two “talks” instead of changing the template?

**Comment by elmiko** [comment]
ok, so it looks like liquid will handle the links properly if we have optional lists

@willb that works too, i'm good either way

File: community.html [hunk 1]
```diff
@@ -72,25 +72,26 @@ <h2> {{ pres.title }}</h2>
         <div class="list-group-item-container container-fluid hidden">
           <div>
             <p><em>{% if pres.venue %}{{ pres.venue }} • {% endif %}{% if pres.city %}{{pres.city}} • {% endif %}{{ pres.date | date: '%B %Y' }}</em></p>
-            
-            {{ pres.content }}
 
-            <a href="{{ pres.media-url }}">
+            {{ pres.content }}
+            {% for murl in pres.media-url %}
```
Comment (erikerlandson) [inline community.html]: how about this: a new media-url-list field.  
if it sees a `pres.media-url-list` it does this looping thing, and if it sees a regular `pres.media-url` it does its current logic

**Comment by elmiko** [comment]
ok, updated so that it doesn't change all the preso. this will allow for option multi-links

File: community.html (hunk 1 above)
Comment (elmiko) [reply community.html]: i think the current version will actually work with lists or single entries, so we should be good

**Comment by erikerlandson** [comment]
LGTM

**Pull request merged**